### PR TITLE
Fixing display of delete confirmation modal

### PR DESCRIPTION
**Pull request by eric-gunther**
Fixes this issue

image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -150,7 +150,6 @@ const components: DemoComponent[] = [
           conditionsOutsideProps={{
             hideRequestRecords: true,
           }}
-          conditionsAllProps={{ onlyAllowAddOutsideConditions: true }}
```
Comment (melocule) [inline src/App.tsx:153]: Wait is this change related to the display in the delete modal? Or is this a separate fix

Comment (eric-gunther) [reply src/App.tsx:153]: Just a minor change to the sample/troubleshooting app so that you actually see the delete button.

**Review by melocule — APPROVED** [review]
